src/components/Header/index.jsx

```diff
@@ -1,27 +1,30 @@
-import React, { PureComponent } from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import { NavLink } from 'react-router-dom';
+import { observer, inject } from 'mobx-react';
 
 import Logo from '../Logo';
 import Input from '../Base/Input';
 import styles from './index.scss';
 
-export default class Header extends PureComponent {
-  static propTypes = {
+@inject('rootStore')
+@observer
+export default class Header extends Component {
+  state = {
     isHome: PropTypes.bool,
   }
 
   render() {
-    const { isHome } = this.props;
-
-    const logoUrl = isHome ? '../../assets/logo.svg' : '../../assets/logo2.svg';
+    const { isHome, rootStore: { fixNav } } = this.props;
+    const isDark = !isHome || fixNav;
+    const logoUrl = isDark ? '../../assets/logo2.svg' : '../../assets/logo.svg';
```

Comment:
``../../assets/${isDark ? "logo2" : "logo"}.svg`` 
Is it better?

Reply:
seems no benefit, and string template will generate new string every time